retorch-llm-rp/pom.xml

```diff
@@ -0,0 +1,80 @@
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.github.giis-uniovi</groupId>
+        <artifactId>retorch</artifactId>
+        <version>1.1.1-SNAPSHOT</version>
+    </parent>
+    <artifactId>retorch-llm-rp</artifactId>
+    <packaging>jar</packaging>
+    <name>retorch-llm-rp</name>
+    <description>Multimodule maven project template, module without dependencies</description>
+    <url>http://github.com/giis-uniovi/samples-giis-template</url>
+    <organization>
+        <name>Software Engineering Research Group (GIIS) - University of Oviedo, ES</name>
+        <url>http://giis.uniovi.es/</url>
+    </organization>
+    <dependencies>
```

Comment:
Either if this is finally a multimodule or not, check for removal all unnecessary stuff (dependencies and build steps) in the pom (or poms)